Decision records/04 - We do not use test IDs in tests.md

```diff
@@ -0,0 +1,49 @@
+# We do not use test IDs in tests
+
+## Date
+
+July 10, 2018
+
+## Contributors
+
+* Chris Sauvé
+* Mallory Allen
+
+## Summary
+
+Test IDs make it too easy to test implementation details of components. They should be avoided in favor of [alternatives](../Best%20practices/Testing.md#test-ids) that emphasize the guarantees a component actually makes.
+
+## Description
+
+Developers sometimes feel the need to add "hooks" that allow their tests to easily find parts of the component subtree and assert details about those subtrees in tests. In the React ecosystem, this is a key recommendation of an alternative to Enzyme (our preferred React testing library), [`react-testing-library`/ `dom-testing-library`](https://github.com/kentcdodds/dom-testing-library/blob/master/README.md#faq).
+
+## History
+
+In building the initial version of Shopify Web, we took several steps to encourage and facilitate the use of test IDs:
+
+* A [babel plugin](https://github.com/lemonmade/babel-plugin-react-test-id) to remove the unnecessary attribute in production
+* Allowing the `testID` prop implicitly with TypeScript and adding a `findByTestId` utility to Shopify Web
+
+These were originally introduced to facilitate differentiation between the same React component rendered multiple times within a tree (specifically, to target multiple `TextField` components rendered by a card in the product section of the admin). No analysis of alternatives was performed at the time. The cases in which this attribute was used expanded and, as of July 10, 2018, it is used 280 times in the Web codebase.
```

Comment:
I don't think you need the part in parens. And maybe `Web` should be `Shopify Web` to match when it is mentioned above.